sinks/src/main/scala/com.snowplowanalytics.snowplow.eventgen/Config.scala

```diff
@@ -14,28 +14,26 @@ package com.snowplowanalytics.snowplow.eventgen
 
 import java.nio.file.Path
 import java.net.URI
+import java.time.Instant
 import scala.io.Source
 import cats.implicits._
-import cats.effect.Sync
 import com.monovore.decline._
-import com.typesafe.config.ConfigFactory
-import io.circe.{Decoder, Json}
+import com.typesafe.config.{Config => RawConfig, ConfigFactory}
+import io.circe.Decoder
+import io.circe.config.parser
 import io.circe.generic.semiauto.deriveDecoder
-import pureconfig._
-import pureconfig.module.circe._
-import pureconfig.error.{CannotParse, ConfigReaderFailures}
-
-import scala.util.{Failure, Success}
 
 final case class Config(payloadsTotal: Int,
                         seed: Long,
                         compress: Boolean,
                         eventPerPayloadMax: Int,
                         eventPerPayloadMin: Int,
                         withRaw: Boolean,
+                        withEnrichedTsv: Boolean,
+                        withEnrichedJson: Boolean,
                         payloadsPerFile: Int,
-                        enrichFormat: Config.EnrichFormat,
                         duplicates: Option[Config.Duplicates],
+                        now: Instant
```

Comment:
I will make it optional.

I really like the idea that the tool generates the exact same output each time you run it, by default.  It seems like a great way to share reference data sets, by sharing a configuration file instead of the data itself..  But I also like the idea of generating "recent" data to feed into a real time pipeline.